src/js/components/Menu/__tests__/__snapshots__/Menu-test.js.snap

```diff
@@ -1401,11 +1401,20 @@ exports[`Menu custom theme with default button 1`] = `
   background: transparent;
   overflow: visible;
   text-transform: none;
-  outline: none;
   border: none;
-  padding: 0;
-  text-align: inherit;
-  color: inherit;
+  border-radius: 18px;
```

Comment:
These changes are expected because `default button` is now being used instead of `plain`